qa/rpc-tests/sc_split.py

```diff
@@ -106,89 +78,98 @@ def run_test(self):
 
         # forward transfer amount
         creation_amount = Decimal("0.5")
-        fwt_amount_1 = Decimal("0.2")
+        fwt_amount_1 = Decimal("4.0")
         fwt_amount_2 = Decimal("1.0")
 
         blocks = [self.nodes[0].getblockhash(0)]
 
         # node 1 earns some coins, they would be available after 100 blocks
-        self.mark_logs("Node 1 generates 1 block")
+        mark_logs("Node 1 generates 1 block",self.nodes,DEBUG_MODE)
 
         blocks.extend(self.nodes[1].generate(1))
         self.sync_all()
 
-        self.mark_logs("Node 0 generates 220 block")
+        mark_logs("Node 0 generates 220 block",self.nodes,DEBUG_MODE)
 
         blocks.extend(self.nodes[0].generate(220))
         self.sync_all()
 
         # Split the network: (0)--(1) / (2)
-        self.mark_logs("\nSplit network")
+        mark_logs("\nSplit network",self.nodes,DEBUG_MODE)
         self.split_network()
-        self.mark_logs("The network is split: 0-1 .. 2")
+        mark_logs("The network is split: 0-1 .. 2",self.nodes,DEBUG_MODE)
 
         txes = []
 
         # ---------------------------------------------------------------------------------------
         # Nodes 1 creates the SC
-        print("------------------------------")
-        self.mark_logs("\nNode 1 creates the SC")
+        mark_logs("\nNode 1 creates the SC",self.nodes,DEBUG_MODE)
 
         amounts = [{"address": "dada", "amount": creation_amount}]
         tx_create = self.nodes[1].sc_create(scid, 123, amounts)
         txes.append(tx_create)
         self.sync_all()
 
+        # Node 0 try create a SC with same id
+        mark_logs("\nNode 0 try creates the same SC",self.nodes,DEBUG_MODE)
+        try:
+            self.nodes[0].sc_create(scid, 123, amounts)
+        except JSONRPCException, e:
+            errorString = e.error['message']
+            mark_logs (errorString,self.nodes,DEBUG_MODE)
+        assert_equal("Transaction commit failed" in errorString, True)
+
+
         # Nodes 1 send 5.0 coins to a valid taddr to verify the network split
-        self.mark_logs("\nNode 1 send 5.0 coins to a valid taddr to verify the network split")
+        mark_logs("\nNode 1 send 5.0 coins to a valid taddr to verify the network split",self.nodes,DEBUG_MODE)
 
         txes.append(self.nodes[1].sendtoaddress("zthXuPst7DVeePf2ZQvodgyMfQCrYf9oVx4", 5.0))
         self.sync_all()
 
         # Check the mempools of every nodes
-        print ("\nChecking mempools...")
+        mark_logs ("\nChecking mempools...",self.nodes,DEBUG_MODE)
         for i in range(0, NUMB_OF_NODES):
             txmem = self.nodes[i].getrawmempool()
             if i == 2:
                 assert_equal(len(txmem), 0)
             else:
                 assert_equal(sorted(txes), sorted(txmem))
```

Comment:
I would move this block up, immediately after network split and before any other sync operation, since you state in comment that the purpose of these lines is to check network split went allright.
As an alternative, you may even remove these lines, since test should be able to use network split function with confidence it works, not retesting it.